src/v2/stores/nodes.js

```diff
@@ -48,7 +48,7 @@ class Store {
   });
 
   get mapMarkers() {
-    if (!this.network || !this.network.length) {
```

Comment:
@sunnygleason network is Proxy object and does not have length property.

Reply:
@manuel-calavera interesting, this code was throwing before, hopefully the try/catch is enough

Here's the previous PR to fix the issue:

https://github.com/solana-labs/networkexplorer/pull/543/files